tests/modules/test_time.py

```diff
@@ -177,6 +176,131 @@ def test_struct_time_repr(self):
 
         self.assertCodeExecution(adjust(test_str))
 
+    # TESTS FOR MKTIME
+    def test_mktime(self):
+        """
+        tests the happy path for the mktime constructor
+        """
+
+        seq = (1970,1,1,0,0,0,0,0,0)
```

Comment:
A small formatting fix: spaces after commas. But something more substantial; it would be a good idea to test something other than "Midnight Jan 1 1970". This test doesn't validate that you're actually using arguments 2 & 3, or 4-9 in the right order - you might, for example, be getting the month and the day the wrong way around (you're not, but hopefully you can see where the *potential* bug might lie).